bundles/org.openhab.binding.bluetooth.daikinmadoka/src/main/java/org/openhab/binding/bluetooth/daikinmadoka/internal/BRC1HUartProcessor.java

```diff
@@ -94,6 +104,7 @@ public void chunkReceived(byte[] byteValue) {
 
             this.responseListener.receivedResponse(bos.toByteArray());
         }
+        stateLock.unlock();
```

Comment:
all locking should be done in a try statement with the unlock inside the `finally` clause.

Reply:
Done